.github/workflows/terragrunt-plan-production.yml

```diff
@@ -51,35 +51,15 @@ jobs:
         with:
           is-tagged: ${{ !startsWith(github.head_ref, 'release-please--') }} # If not the release PR branch, assume it is a revert PR
 
-  detect-lambda-changes:
-    needs: get-version
-    runs-on: ubuntu-latest
-    env:
-      VERSION: ${{ needs.get-version.outputs.version }}  
-    outputs:
-      lambda-to-rebuild: ${{ steps.filter.outputs.changes }}
-    steps:
-      - name: Checkout
-        uses: actions/checkout@a5ac7e51b41094c92402da3b24376905380afc29 # v4.1.6
-        with:
-          ref: ${{ env.VERSION }}
-
-      - name: Filter
-        id: filter
-        uses: dorny/paths-filter@de90cc6fb38fc0963ad72b210f1f284cd68cea36 # v3.0.2
-        with:
-          filters: .github/lambda-filter.yml
-
   test-lambda-code:
-    needs: [get-version, detect-lambda-changes]
-    if: needs.detect-lambda-changes.outputs.lambda-to-rebuild != '[]'
+    needs: get-version
     runs-on: ubuntu-latest
     env:
       VERSION: ${{ needs.get-version.outputs.version }}
     strategy:
       fail-fast: false
       matrix:
-        image: ${{ fromJSON(needs.detect-lambda-changes.outputs.lambda-to-rebuild) }}
+        image: [audit-logs, audit-logs-archiver, cognito-email-sender, cognito-pre-sign-up, form-archiver, nagware, notify-slack, reliability, reliability-dlq-consumer, response-archiver, submission, vault-integrity]
```

Comment:
I've seen this before. Was it done for staging but not for production?

Reply:
No, it was done for the TF apply prod workflow because of the same bug.  It's on our list to generate this list dynamically rather than hardcoding it in both workflows.